apps/designer/package.json

```diff
@@ -13,7 +13,8 @@
     "watch:sdk": "relative-deps && relative-deps watch",
     "prettier": "prettier \"*.{js,md,json,ts}\" \"{docs,app}/**/*.{js,md,json,ts}\"",
     "storybook:run": "start-storybook -p 6006",
-    "storybook:build": "build-storybook"
+    "storybook:build": "build-storybook",
+    "migrations": "migrations --dev"
```

Comment:
@kof For working with migrations during development it's currently set up like this:

```
$ cd apps/designer
$ yarn migrations # will output usage information
$ yarn migrations command # will run a command 
```